importer_jenkins_setup.sh

```diff
@@ -61,7 +61,12 @@ pip install -r metadata_tools/requirements.txt
 
 # creating databases
 
-docker run --name mariadb-specify -e MARIADB_ROOT_PASSWORD=password -d -p 3310:3306 mariadb:latest
+docker run --name mariadb-specify \
+  -e MARIADB_ROOT_PASSWORD=password \
+  -e MARIADB_CHARSET=utf8mb4 \
+  -e MARIADB_COLLATION=utf8mb4_general_ci \
+  -d -p 3310:3306 mariadb:10.11
```

Comment:
with versions of mariadb 11.0 and higher, (we have 10.11 in prod right now) assumes a collation of utf8mb4_0900_ai_ci when used with mysql-python-connector , while version lower use utf8mb4_ci_general